staging/src/k8s.io/apiserver/pkg/util/openstack/types.go

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright 2021 The Arktos Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package openstack
+
+import (
+	"strconv"
+
+	v12 "k8s.io/api/apps/v1"
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/json"
+)
+
+const (
+	LABEL_SELECTOR_NAME = "ln"
+	DEFAULT_NAMESPACE   = "kube-system"
+	DEFAULT_TENANT      = "system"
```

Comment:
There should be default tenant system constant defined somewhere else.

Reply:
same as above. so resolve to use the previous one to track potential fixes.

Reply:
resolve as same reasoning as for the namespace string